ajax/libs/strapdown-topbar/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "strapdown-topbar",
+  "filename": "strapdown-topbar.min.js",
+  "version": "1.6.3",
+  "description": "a topbar modification for strapdown.js",
+  "homepage": "http://joedf.github.io/strapdown-topbar",
```

Comment:
@joedf I think it'll be better to use `https` for `github.io` domain. How do you think?